section_names.yml

```
@@ -11,4 +11,5 @@ section_names:
   domain specific libraries: Domain-specific libraries
   documentation: Documentation
   experimental: Experimental projects that may be merged upstream eventually
+  ides: IDEs with built-in interactive backends
```

I am not sure many folks will know what all the jargon means there.  Maybe:

```suggestion
  ides: Development environments with interactive Matplotlib figures
```

---

"Editors"?

I agree that it may not be fully correct, but to me "Development environments" can sound a bit like more than the IDE/Editor. Then, is VS Code an editor or an IDE? When does it become an IDE?

I wonder if "interactive" is a thing here? Spyder is, out of the box, basically rendering the plots in a dedicated tab, without any interactivity. Again, maybe not fully correct, but "Editors with dedicated Matplotlib support", "Editors with built-in Matplotlib rendering"?

---

"Rendering backends" is similarly un-informative if you don't know the jargon, but OK there, because if you don't know what a backend is, you probably shouldn't be using an alternative.  

I like "Editors with built-in Matplotlib rendering" or similar.